Configuration.props

```
@@ -75,13 +75,19 @@
     <DotNetPreviewPath Condition=" '$(DotNetPreviewPath)' == '' ">$(AndroidToolchainDirectory)\dotnet\</DotNetPreviewPath>
     <DotNetPreviewTool Condition=" '$(DotNetPreviewTool)' == '' ">$(DotNetPreviewPath)dotnet</DotNetPreviewTool>
     <!-- Version number from: https://github.com/dotnet/installer#installers-and-binaries -->
+    <!-- Please update DotNetRuntimePacksVersion below accordingly -->
     <DotNetPreviewVersionBand Condition=" '$(DotNetPreviewVersionBand)' == '' ">6.0.100</DotNetPreviewVersionBand>
     <DotNetPreviewVersionFull Condition=" '$(DotNetPreviewVersionFull)' == '' ">$(DotNetPreviewVersionBand)-preview.1.21103.13</DotNetPreviewVersionFull>
+    <!-- This version comes from the a file in the dotnet distribution: sdk/$(DotNetPreviewVersionFull)/dotnet.runtimeconfig.json -->
+    <DotNetRuntimePacksVersion Condition=" '$(DotNetRuntimePacksVersion)' == '' ">[6.0.0-preview.2.21080.7]</DotNetRuntimePacksVersion>
```

I think you could do:

```suggestion
    <DotNetRuntimePacksVersion Condition=" '$(DotNetRuntimePacksVersion)' == '' ">6.0.0-preview.2.21080.7</DotNetRuntimePacksVersion>
```

If you put the `[]` in the `@(PackageDownload)`, you wouldn't have to trim these `[]` in other places. This worked for me [here](https://github.com/jonathanpeppers/maui-workload/blob/ec0530198cdf17f7e2dba89807f6282afb933922/src/Maui.Dependencies/Maui.Dependencies.Sdks.csproj#L6-L11):

```diff
<PackageDownload Include="Microsoft.NETCore.App.Runtime.android-arm64" Version="[$(DotNetRuntimePacksVersion)]" />
```

It also kind of makes sense to put the weird NuGet requirement in this one spot.